README.rst

```diff
@@ -11,18 +11,18 @@ Ansible Molecule
    :alt: Documentation Status
 
 .. image:: https://github.com/ansible-community/molecule/workflows/tox/badge.svg
```

Comment:
You cannot use spinx tricks in this file, it will break rendering on pypi as that goes into package metadata, not even reaching sphinx.

Reply:
I'll revert that.

And I just found https://github.com/sphinx-doc/sphinx/issues/10113, so I guess building docs locally will be broken for a bit longer...

While I'm at it, is there anything wrong with adding the `--keep-going` parameter to the doc building command in the tox.ini? Building the docs would still fail, but it would print out all the errors and not just the first one. And then you go fix it and it spits out the next one.